### ISSUE TITLE
About loss in Tensorboard

### ISSUE THREAD

**Issue by zhao62** [tflearn/tflearn#882]
Hello everyone,

I run the example of Multi-layer perceptron, and visualize the loss in Tensorboard.

Does "Loss" refer to the training loss on each batch? And "Loss/Validation" refers to the loss on validation set? What does "Loss_var_loss" refer to?

![screenshot from 2017-08-22 10-49-05](https://user-images.githubusercontent.com/30203331/29571631-b4c9b48a-8727-11e7-98ba-0d6ed9dc1c86.png)

**Comment by zhao62** [comment]
Does anyone know this? Thanks in advance.

**Comment by Noahyt** [comment]
"loss" refers to loss on training set .  "loss /validation" is just loss on validation set (don't know why its so poorly labeled).  I am not sure what Loss_var_loss refers to

**Comment by zhao62** [comment]
Thank you very much.

I have one more question. Does "loss" refer to the loss on the total training set? Or just the loss on the mini-batches in the training process? 

I guess the "loss" should be more smoothed if it is on the total training set.
